### PR TITLE
USHIFT-1118: Merge Router and Load-Balancer suite into single Networking

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: test/resources/oc.resource [hunk 1]
```diff
@@ -59,3 +59,10 @@ Oc Create
     [Arguments]    ${opts}
     ${output}=    Run With Kubeconfig    oc create ${opts}
     RETURN    ${output}
+
+Oc Expose
+    [Documentation]    Run 'oc expose' on a specific resource in the curret test namespace
+    ...    Returns the command's combined STDOUT/STDER
```
Comment (ggiguash) [inline test/resources/oc.resource:65]:
```suggestion
    [Documentation]    Run 'oc expose' on a specific resource in the current test namespace
    ...    Returns the combined STDOUT/STDERR of the command output.
```

@eslutsky , can you fix the same typos in the `Oc Create` as well?

File: test/suites/standard/networking-smoke.robot [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+*** Test Cases ***
+Router Smoke Test
+    [Documentation]    Run a router smoke test
+    [Setup]    Run Keywords
+    ...    Create Hello MicroShift Pod
+    ...    AND
+    ...    Expose Hello MicroShift Service Via Route
+
+    Wait Until Keyword Succeeds    10x    6s
+    ...    Access Hello Microshift    80
```
Comment (ggiguash) [inline test/suites/standard/networking-smoke.robot:28]: Should we define `80` port in HTTP_PORT variable?

File: test/suites/standard/networking-smoke.robot [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+*** Keywords ***
+Create Hello MicroShift Pod
+    [Documentation]    Create a pod running the "hello microshift" application
+    Oc Create    -f ${HELLO_USHIFT} -n ${NAMESPACE}
+    Oc Wait For    pods -l app=hello-microshift    condition\=Ready    timeout=60s
+
+Expose Hello MicroShift Pod Via LB
+    [Documentation]    Expose the "hello microshift" application through the load balancer
+    Run With Kubeconfig    oc create service loadbalancer hello-microshift --tcp=5678:8080 -n ${NAMESPACE}
```
Comment (ggiguash) [inline test/suites/standard/networking-smoke.robot:56]:
```suggestion
    Run With Kubeconfig    oc create service loadbalancer hello-microshift --tcp=${LB_PORT}:8080 -n ${NAMESPACE}
```

File: test/suites/standard/networking-smoke.robot [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    ${result}=    Run Process
+    ...    curl -i http://hello-microshift.cluster.local --connect-to ${connect_to}
+    ...    shell=True
+    ...    timeout=15s
+    Log Many    ${result.rc}    ${result.stdout}    ${result.stderr}
+    Should Be Equal As Integers    ${result.rc}    0
+    Should Match Regexp    ${result.stdout}    HTTP.*200
+    Should Match    ${result.stdout}    *Hello MicroShift*
+
+Access Hello Microshift Via Route
```
Comment (ggiguash) [inline test/suites/standard/networking-smoke.robot:78]: This function and `Access Hello MicroShift` seem to be the same except for the port argument. Can we always use `Access Hello MicroShift` with the appropriate port args?

Comment (eslutsky) [reply test/suites/standard/networking-smoke.robot:78]: yes, this Keyword is not used anymore, removing

**Pull request merged**